{
  File cassandraDslFile=new File(CORRUPTED_RESOURCES_PATH + "cassandra_name_field_corrupted-service.groovy");
  File cassandraWorkDir=new File(CORRUPTED_RESOURCES_PATH);
  try {
    Service service=ServiceReader.getServiceFromFile(cassandraDslFile,cassandraWorkDir).getService();
    assertTrue("No exception was thrown due to Name field corruption",false);
  }
 catch (  Throwable t) {
    assertTrue("Throwable isn't informative enought",t.getMessage().toLowerCase().contains("nae"));
  }
}
